src/tests/.Net 7.0/Microsoft.PowerFx.Connectors.Tests/Microsoft.PowerFx.Connectors.Tests.csproj

```diff
@@ -41,4 +40,8 @@
 
   <Import Project="..\..\Microsoft.PowerFx.Connectors.Tests.Shared\Microsoft.PowerFx.Connectors.Tests.Shared.projitems" Label="Shared" />
 
+  <ItemGroup>
+    <Compile Update="C:\Data\3\Power-Fx\src\tests\Microsoft.PowerFx.Connectors.Tests.Shared\AssemblyProperties.cs" Link="AssemblyProperties.cs" />
```

Comment:
>"C:\Data\3\ [](http://example.com/codeflow?start=20&length=11)

?

Reply:
VS can be creative at times
I'll remove for sure